redbot/core/cli.py

```diff
@@ -74,6 +74,18 @@ def confirm(text: str, default: Optional[bool] = None) -> bool:
     return token
 
 
+def positive_int(arg: str) -> int:
+    try:
+        x = int(arg)
+    except ValueError:
+        raise argparse.ArgumentTypeError("Message cache size has to be a number.")
+    if x <= 0:
```

Comment:
After discussion in Discord, we decided that the message cache size shouldn't be allowed to be lower than 1k.
`positive_int` function should have a different name and it should check for `x < 1000` instead here.